PatTools/plugins/MiniAODJetIdEmbedder.cc

```diff
@@ -125,8 +125,9 @@ void MiniAODJetIdEmbedder::produce(edm::Event& evt, const edm::EventSetup& es) {
       }
 
     jet.addUserFloat("puID", float(passPU));
-
-    output->push_back(jet);
+    if (loose == true){
```

Comment:
What is this doing? we don't want to remove jets here... that is done at the cleaning step. This is an embedder.